### PR TITLE
Feature/dp 5861 state2url

### PR DESCRIPTION
**Pull request by aditudorache**

### CODE REVIEW

File: src/shared/environment.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ export const isDevelopment = () => {
 
 // This object enables/disables features that are not yet released
 export const features = {
-  eigendommen: false
+  eigendommen: true
```
Comment (chrisvanmook) [inline src/shared/environment.js]: Should this be enabled?

File: src/shared/services/routing/state-to-url.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    const params = stateUrlConverter.state2params(state);
+    const hash = getSanitizedHash(params);
+
+    if (!isHashChanged(location.hash, hash)) return;
+
+    // eslint-disable-next-line no-console
+    // console.log('updating url hash (location.hash and hash are different)');
+    if (useReplace) {
+      console.log('replace', hash); // eslint-disable-line no-console
```
Comment (chrisvanmook) [inline src/shared/services/routing/state-to-url.js]: Do we really need console.log here?

File: src/shared/environment.js (hunk 1 above)
Comment (aditudorache) [reply src/shared/environment.js]: yes, we test all the existing features here, including eigendommen

File: src/shared/services/routing/state-to-url.js (hunk 2 above)
Comment (aditudorache) [reply src/shared/services/routing/state-to-url.js]: removed

**Pull request merged**